packages/cra-template/.eslintrc.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/node"],
```

Probably use the `react` config for this (though there's not a significant difference at the moment)
```suggestion
  "extends": ["plugin:@fluentui/eslint-plugin/react"],
```

---

Will do, thanks! Point to remember, this was auto-generated by `create-package` script, so perhaps this needs some refactoring as well.

---

I think this type of package is a special case where it would be hard for create-package to know the "right" thing to do. It's not a component package (the main intent of the "react" package template currently), but it's also not a build tool exactly. We could potentially add some more refinement to the template options, but I've had trouble figuring out the best way to structure that. (Also there's an active PR for create-package right now, so if you wanted to change anything it would be best to wait until that's done.)